codecov.yml

```diff
@@ -0,0 +1,18 @@
+comment: false
```

Comment:
So how will this setup work? I see a comment on this PR. But maybe that was before you've set `comment: false`. Does this mean no comments? And if so, how will we know the coverage of a patch?

It would be nice if it would show up as one of the checks in the PR. How can we do that? It's possible because I can see it on their homepage.

Reply:
`comment:false` would make sure the Codecov comments don't show up in PRs.